src/lib/JSONEditor.js

```diff
@@ -136,7 +138,7 @@ export default class JSONEditor extends React.Component {
             value={label}
             addElement={this.addElement}
             removeElement={this.removeElement}
-            showRemoveButton={this.props.showRemoveButton}
+          showRemoveButton={this.props.showRemoveButton}
```

Comment:
Please Check indentations!

Reply:
Thanks Nikhil, The indentations are now taken care of.